rasa/utils/train_utils.py

```diff
@@ -37,19 +40,38 @@
     from rasa.nlu.tokenizers.tokenizer import Token
 
 
-def normalize(values: np.ndarray, ranking_length: Optional[int] = 0) -> np.ndarray:
+def normalize(values: np.ndarray) -> np.ndarray:
     """Normalizes an array of positive numbers over the top `ranking_length` values.
 
+    Args:
+        values: Values to normalize
+
+    Returns:
+        Normalized values.
+    """
+    new_values = values.copy()
+
+    if np.sum(new_values) > 0:
+        new_values = new_values / np.sum(new_values)
+
+    return new_values
+
+
+def filter_top_k(values: np.ndarray, ranking_length: Optional[int] = 0) -> np.ndarray:
```

Comment:
there is no need for this method as a separate, the filtering was done only for normalization